RadarSDK/Radar.m

```diff
@@ -859,4 +860,54 @@ - (void)dealloc {
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
++ (BOOL)isTestKey {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if ([publishableKey hasPrefix:@"prj_test_pk"] ||
+        [publishableKey hasPrefix:@"org_test_pk"]) {
+        return YES;
+    }
+    return NO;
+}
+
++ (void)sendLog:(RadarLogLevel)level message:(NSString *_Nonnull)message {
+    [[RadarLogBuffer sharedInstance] write:level message:message];
+}
+
+/**
+ * Sends Radar log events to the server
+ */
++ (void)flushLogs {
+    // user _id has to exist
+
+    if (![self isTestKey]) {
+        return;
+    }
+
+    NSArray<RadarLog *> *flushableLogs = [[RadarLogBuffer sharedInstance] flushableLogs];
```

Comment:
I kind of wonder if it's the responsibility of the caller to care about specifying log counts to flush vs just initializing that with the `RadarLogBuffer` to begin with. 

I guess it's a shared instance, but it seems a bit finicky to have to do all these checks / multiple reads (thinking of concurrency as well) from the call-side. does it even matter if the `pendingLogCount` is `0`? couldn't the buffer automagically handle that as well?

Reply:
even the API logic feels like it could live in the RadarLogBuffer, or be passed a callback function to process the logs.

Reply:
I agree that it shouldn't be the caller's responsibility, but maybe that can be addressed in a future PR.

Reply:
yea i agree some of this logic can shift to the buffer's responsibility, but trying to stay close to android
(check for test key, skip if log count is 0, handle adding logs back to buffer on error)
https://github.com/radarlabs/radar-sdk-android/blob/master/sdk/src/main/java/io/radar/sdk/Radar.kt#L2182

i think for the log count though it should be here, since if no logs we want to skip the api call

Reply:
yeah, I think an implementation of something like:

```
[[RadarLogBuffer sharedInstance] flushLogs:maxLogsCount callback:sendLogsToAPI]
```

where X is the API caller which gets passed a list of logs, might make this a little cleaner. anyways this current impl is fine